### DISCUSSION TITLE
Conversation on mobile - Pathway to "HER"

### DISCUSSION THREAD

**Discussion by HappyCatKitten** [oobabooga/text-generation-webui#179] [Ideas]
Would be cool to eventually have the following

Thin mobile client that can stream audio between your server
Whisper 
LLM chat
TTS with real emotion

**Comment by oobabooga** [comment]
I guess that would be the chat mode endgame.

**Comment by pbosh** [comment]
Eleven Labs is currently the best TTS for emotion. I also used Sonantic for a year or so (even in some commercial projects) until they got bought by Spotify.

**Comment by oobabooga** [comment]
There is an elevenlabs extension already: https://github.com/oobabooga/text-generation-webui/wiki/Extensions#examples

There is also a whisper extension in the works: https://github.com/oobabooga/text-generation-webui/pull/207

And a PR for autoplaying the TTS outputs: https://github.com/oobabooga/text-generation-webui/pull/192
